GOVERNANCE.md

```diff
@@ -7,12 +7,25 @@ This document defines project governance for the Strimzi project.
 ### Maintainers
 
 The maintainers are responsible for the overall development of the whole Strimzi project and all its components.
-The maintainers are identified in the [`MAINTAINERS`](MAINTAINERS) file. 
+Emeritus maintainers are former maintainers who are no longer actively involved in the project.
+In the rest of this document the term ‘maintainer’ will be assumed to mean ‘active maintainer’, unless qualified as ‘emeritus maintainer’.
+Both maintainers and emeritus maintainers are identified in the [`MAINTAINERS`](MAINTAINERS) file. 
 
 #### Changes in Maintainership
 
 New maintainers are proposed by an existing maintainer and are elected by a ⅔ majority maintainers vote.
-Maintainers can be removed by a ⅔ majority maintainers vote.
+Maintainers can be removed by a ⅔ majority maintainers vote. 
+Maintainers removed by a vote are no longer considered maintainers or emeritus maintainers.
+Maintainers become emeritus maintainers either actively, by opening a PR against the MAINTAINERS file marking themselves as Emeritus, or implicitly after 6 months without any contribution to the project.
+
+### Emeritus maintainers
+
+An Emeritus maintainer has still access to all the Strimzi venues (e.g. Slack, Email, Community call, etc.).
+They can contribute to the project in any way which doesn't implies strong and long term commitment, as active maintainers do.
+The MAINTAINERS file may not always immediately reflect implicit emeritus status, as updates to mark maintainers as Emeritus are done asynchronously.
+An Emeritus maintainer reverts to being an active maintainer by starting a voting process.
+The voting process can also be started by an active maintainer instead.
```

Comment:
I think it would be weird to have an active maintainer try to reactivate an emeritus maintainer like this. 

Firstly, it seems unnecessary because there's nothing stopping the EM starting the vote themselves. 

Secondly, I think it's something they should explicitly consent to. They might not _want_ to become active, so no one should try to force them. Note that when someone is proposed for maintainer (originally) it results in an _offer_, which they can refuse, so for that part of the process there is a consent element already. It seems odd not to have it here too.